src/Stratis.Features.FederatedPeg/TargetChain/WithdrawalTransactionBuilder.cs

```diff
@@ -73,9 +73,11 @@ public Transaction BuildWithdrawalTransaction(uint256 depositId, uint blockTime,
             }
             catch (Exception error)
             {
-                if (error is WalletException walletException && walletException.Message == FederationWalletTransactionHandler.NoSpendableTransactionsMessage)
+                if (error is WalletException walletException && 
```

Comment:
Won't block as it's outside scope, but would be nice if we could change this to `catch (WalletException)` and use an exception filter for the message.